Dockerfile

```diff
@@ -20,6 +20,8 @@ RUN npm --unsafe-perm install --only production
 # Setup the application code
 COPY src /opt/project/src
 
+RUN npm update auth-middleware
```

Comment:
I'd prefer pinning this so that dependencies are stable. I understand that we are still developing this, but once this is working on dev we should tag and point the dependency to the tag.

Reply:
@andres-arana I aggree with you. This is only provisional.